infra/terraform/external_function/snowalert.tf

```diff
@@ -25,14 +34,48 @@ variable "snowflake_integration_user" {
   default     = null
 }
 
-variable "snowflake_integration_external_id" {
-  type        = string
-  description = "API_AWS_EXTERNAL_ID from DESC INTEGRATION ..."
-  default     = "000000"
-}
-
 variable "aws_cloudwatch_metric_namespace" {
   type        = string
   description = "where EF can write CloudWatch Metrics"
   default     = "*"
 }
+
+variable "gateway_logger_name" {
+  type        = string
+  default     = "gateway_logger1"
+}
+
+variable "gateway_caller_name" {

```

Comment:
same as above -- `aws_gateway_caller_iam_role_name` and a description like "IAM Role the Snowflake user will assume to call the API Gateway in your account".